templates/BUILD.tpl

```diff
@@ -143,7 +145,11 @@ filegroup(
 
 filegroup(
     name = "compiler_deps",
-    srcs = glob(["extra_tools/**"], allow_empty = True),
```

Comment:
Is this always empty or should we keep it?

Reply:
it's always empty so I removed it.